### PR TITLE
Ignore E2E CI tests on Windows

### PR DESCRIPTION
**Pull request by farmaazon**
### Pull Request Description

The integration tests are constantly failing on Windows. For the time of investigation, they should be suppressed.

### CODE REVIEW

**Review by vitvakatu — APPROVED** [review]

File: .github/workflows/gui.yml [hunk 1]
```diff
@@ -512,6 +512,7 @@ jobs:
           ENSO_TEST_USER: ${{ secrets.ENSO_CLOUD_TEST_ACCOUNT_USERNAME }}
           ENSO_TEST_USER_PASSWORD: ${{ secrets.ENSO_CLOUD_TEST_ACCOUNT_PASSWORD }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        continue-on-error: true
```
Comment (JaroslavTulach) [inline .github/workflows/gui.yml:515]: The PR is called

> Block integration tests

Shouldn't it be _disable_, _disarm_, _ignore results of_?

As right now I am "blocked by integration tests", and I'd like to be _"unblocked"_.

File: .github/workflows/gui.yml [hunk 2]
```diff
@@ -512,6 +512,7 @@ jobs:
           ENSO_TEST_USER: ${{ secrets.ENSO_CLOUD_TEST_ACCOUNT_USERNAME }}
```
Comment (JaroslavTulach) [inline .github/workflows/gui.yml:512]: Ignoring results of the tests is the right fix for now. To quote [my note in another PR](https://github.com/enso-org/enso/pull/11594/commits/383e94195ce024609fdc7835325f30c9bd57d5c0#r1849946447):

This file is labeled with

```yaml
 # This file is auto-generated. Do not edit it manually!
# Edit the enso_build::ci_gen module instead and run `cargo run --package enso-build-ci-gen`.
```

that's why I believe it is a **good idea** to remove this section manually. Next time someone tries to run `cargo run --package enso-build-ci-gen` this section will be regenerated back.

A subsequent PR should fix the tests and execute `cargo run --package enso-build-ci-gen` to enable them again.

File: build/build/src/ci_gen/job.rs [hunk 3]
```diff
@@ -546,13 +546,18 @@ impl JobArchetype for PackageIde {
...
+            let mut test_step = test_step
                 .with_env("DEBUG", "pw:browser log:")
                 .with_secret_exposed_as(secret::ENSO_CLOUD_TEST_ACCOUNT_USERNAME, "ENSO_TEST_USER")
                 .with_secret_exposed_as(
                     secret::ENSO_CLOUD_TEST_ACCOUNT_PASSWORD,
                     "ENSO_TEST_USER_PASSWORD",
                 );
+            // Make E2E tests optional on Windows, as we have an ongoing issue with the runner.
+            // TODO[ib]: remove once the issue is resolved.
+            if target.0 == OS::Windows {
```
Comment (JaroslavTulach) [inline build/build/src/ci_gen/job.rs:558]: Isn't:
```yaml
continue-on-error: true
```
in the `gui.yaml` file a sufficient fix?

**Review by JaroslavTulach — APPROVED** [review]
Disabling the tests shall unlock the CI. Please integrate, if it gets green!

File: .github/workflows/gui.yml (hunk 1 above)
Comment (vitvakatu) [reply .github/workflows/gui.yml:515]: Sure, I changed the name.

File: .github/workflows/gui.yml (hunk 2 above)
Comment (vitvakatu) [reply .github/workflows/gui.yml:512]: I don’t believe changing auto-generated files and introducing a hidden discrepancy between the code and workflow is justified here. To make an example:
- if we have _two_ different CI issues, and we ~~fix~~ mitigate both of them as you are suggesting
- I fix one of them and rerun the generator as suggested
- I suddenly also disabled mitigation of the second issue I’m not even aware of

File: build/build/src/ci_gen/job.rs (hunk 3 above)
Comment (Frizi) [reply build/build/src/ci_gen/job.rs:558]: This is the generator source that outputs `guy.yml` file.

**Review by Frizi — APPROVED** [review]

File: build/build/src/ci_gen/job.rs (hunk 3 above)
Comment (JaroslavTulach) [reply build/build/src/ci_gen/job.rs:558]: I see. I was blind.

**Pull request merged**